crates/fumola/src/lib/vm_def.rs

```diff
@@ -434,7 +438,16 @@ pub mod def {
                 }
                 Ok(())
             }
-            Dec::Type(_id, _typ_binds, _typ) => Ok(()),
+            Dec::Type(_id, _typ_binds, _typ) => {
```

Comment:
In an unrelated effort to this PR, this change slipped in.

It's related to expanding the `#[derive(XYZ)]` annotations that can appear on types in Fumola.  The idea is to expand these into Fumola/Motoko functions that implement the named "traits".